src/xGadgetron/pGadgetron/Gadgetron.py

```diff
@@ -962,8 +962,8 @@ def dimensions(self):
         if self.number() < 1:
             return numpy.zeros((MAX_ACQ_DIMENSIONS,), dtype=cpp_int_dtype())
         dim = numpy.ones((MAX_ACQ_DIMENSIONS,), dtype=cpp_int_dtype())
-        hv = pygadgetron.cGT_getAcquisitionDataDimensions\
-             (self.handle, dim.ctypes.data)
+        hv = try_calling(pygadgetron.cGT_getAcquisitionDataDimensions\
```

Comment:
`try_calling` has no return value (so the next line is no longer needed) - see `src/common/Utilities.py`

Reply:
to be sure, by the next line I mean line 967 (`try_calling` takes care about the handles returned by C interface functions)